install_ibmi.sh

```diff
@@ -37,5 +37,7 @@ mkdir -p source
 cd source
 
 # Clone source code from github repo
-git clone git@github.ibm.com:project-pim/pim.git
-cd pim
+curl "https://codeload.github.com/IBM/project-pim/zip/refs/heads/main" --output pim.zip
+unzip pim.zip
+cd project-pim-main
+export PYTHONPATH=.
```

Comment:
maynot needed here since we run launch cmd outside of installer script?

Reply:
not on the same session where install_ibmi.sh ran, if its a fresh session user needs to set it I believe, right?

Reply:
ok